http_service/bugbug_http/app.py

```diff
@@ -190,6 +190,43 @@ def schedule_job(
     )
 
 
+def prepare_queue_job(job: JobInfo, job_id: Optional[str] = None) -> Queue:
+    """Prepare queue data for for enqueueing multiple jobs"""
+    job_id = job_id or get_job_id()
+
+    redis_conn.mset({job.mapping_key: job_id})
+    return Queue.prepare_data(
+        job.func,
+        *job.args,
+        job_id=job_id,
+        ttl=QUEUE_TIMEOUT,
+        failure_ttl=FAILURE_TTL,
+    )
+
+
+def schedule_multiple_job(jobList: list[Queue] = None) -> None:
+    """Enqueue multiple jobs in bulk"""
+    jobs = q.enqueue_many(jobList)
+
+
+def prepare_multi_bug_classification(model_name: str, bug_ids: Sequence[int]) -> Queue:
+    """Prepare the classification of a bug_id list"""
+    job_id = get_job_id()
+
+    # Set the mapping before queuing to avoid some race conditions
+    job_id_mapping = {}
+    for bug_id in bug_ids:
+        key = JobInfo(classify_bug, model_name, bug_id).mapping_key
+        job_id_mapping[key] = job_id
+
+    redis_conn.mset(job_id_mapping)
+
+    return prepare_queue_job(
+        JobInfo(classify_bug, model_name, bug_ids, BUGZILLA_TOKEN),
+        job_id=job_id,
+    )
+
+
 def schedule_bug_classification(model_name: str, bug_ids: Sequence[int]) -> None:
```

Comment:
Can you refactor schedule_bug_classification and prepare_multi_bug_classification to share the common code?

Reply:
Have refactored schedule_bug_classification method to share the common code.